src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html

```diff
@@ -43,7 +43,7 @@ <h5>{{ cloneHeadline | artemisTranslate }}</h5>
             <pre
                 class="clone-url"
                 [ngClass]="{
-                    'url-box-remove-line-left': sshEnabled && ((useSsh && !localVCEnabled) || !useSsh),
+                    'url-box-remove-line-left': sshEnabled || (useSsh && !localVCEnabled) || !useSsh,
```

Comment:
IMO there is only one way the left side should be cut off: if the SSH/HTTPS selector is visible. This is the case if `sshEnabled` is true.

(Also: `sshEnabled || ... || !useSsh` looks like it's always true - if SSH is disabled, wouldn't `useSsh` always be false?)

I changed this to just `sshEnabled` in #8920

```suggestion
                    'url-box-remove-line-left': sshEnabled,
```